.github/workflows/pyth-sdk.yml

```diff
@@ -25,3 +25,27 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose
+  lints:
+    name: Lints
```

Comment:
this name doesn't seem accurate (?) I think this CI test should be called something like "Check schema changes are committed"

Reply:
yes you are right.